billing.go

```diff
@@ -0,0 +1,165 @@
+	UpdatedAt                int64   `json:"updated_at"`
+	IsRemovable              *bool   `json:"is_removable"`
+}
+
+// BillingSubscriptionItemNextPayment represents next payment info.
+type BillingSubscriptionItemNextPayment struct {
+	APIResource
+
+	Amount *BillingMoney `json:"amount"`
+	Date   *int64        `json:"date"`
+}
+
+// Payer represents a billing payer (user or organization).
+type Payer struct {
+	APIResource
+
+	Object     string `json:"object"`
+	ID         string `json:"id"`
+	InstanceID string `json:"instance_id"`
+
+	// User payer only
+	UserID    *string `json:"user_id"`
+	FirstName *string `json:"first_name"`
+	LastName  *string `json:"last_name"`
+	Email     *string `json:"email"`
+
+	// Org payer only
+	OrganizationID   *string `json:"organization_id"`
+	OrganizationName *string `json:"organization_name"`
+
+	// Used for both org and user payers
+	ImageURL string `json:"image_url"`
+
+	CreatedAt int64 `json:"created_at"`
+	UpdatedAt int64 `json:"updated_at"`
+}
+
+// SubscriptionItem represents a billing subscription item.
+type SubscriptionItem struct {
+	APIResource
+
+	Object          string                              `json:"object"`
+	ID              string                              `json:"id"`
+	InstanceID      string                              `json:"instance_id"`
+	Status          string                              `json:"status"`
+	PlanID          *string                             `json:"plan_id"`
+	Plan            *Plan                               `json:"plan"`
+	PlanPeriod      string                              `json:"plan_period"`
+	PaymentMethodID string                              `json:"payment_method_id"`
+	PaymentMethod   *BillingPaymentMethod               `json:"payment_method"`
+	LifetimePaid    *BillingMoney                       `json:"lifetime_paid"`
+	Amount          *BillingMoney                       `json:"amount"`
+	NextPayment     *BillingSubscriptionItemNextPayment `json:"next_payment"`
+	PayerID         string                              `json:"payer_id"`
+	Payer           *Payer                              `json:"payer"`
+	IsFreeTrial     bool                                `json:"is_free_trial"`
+	PeriodStart     *int64                              `json:"period_start"`
+	PeriodEnd       *int64                              `json:"period_end"`
+	ProrationDate   string                              `json:"proration_date"`
+	CanceledAt      *int64                              `json:"canceled_at"`
+	PastDueAt       *int64                              `json:"past_due_at"`
+	EndedAt         *int64                              `json:"ended_at"`
+	CreatedAt       int64                               `json:"created_at"`
+	UpdatedAt       int64                               `json:"updated_at"`
+}
+
+// SubscriptionItemList contains a list of subscription items.
+type SubscriptionItemList struct {
+	APIResource
+
+	Data       []*SubscriptionItem `json:"data"`
```

Comment:
```suggestion
	Data       []SubscriptionItem `json:"data"`
```

We won't include nil subscription items in responses, right?